getting-started/installation-windows-env.rst

```diff
@@ -44,11 +44,11 @@ If you have Administrator rights and want to do it globally, you can also select
 
 1. Select ``Advance`` tab, navigate to ``Environment Variables`` button. That should open editable tables of settings for Windows Environment. 
 2. Select ``User Variable for 'user'`` option, press NEW
-3. Add EPICS BASE path here. In ``Variable Name``, put "EPICS_BASE". For ``Variable Value``, enter the location of your EPICS Base installation, e.g.,  "C:\\msys64\\home\\'user'\\base-R7.0.4.1"
+3. Add EPICS BASE path here. In ``Variable Name``, put "EPICS_BASE". For ``Variable Value``, enter the location of your EPICS Base installation, e.g.,  "c:\\msys64\\home\\ 'user'\\base-R7.0.4.1"
 4. Set the host architecture. In ``Variable Name``, put "EPICS_HOST_ARCH". For ``Variable Value``, put "windows-x64-mingw" or "windows-x64" (depending on your selection of compilers).
 5. Navigate to the variable called ``Path``. Press Edit. 
-6. If you are using the MinGW compilers and dynamic (DLL) linking, add the path for the MinGW64 DLLs. Press NEW and enter "C:\\msys64\\mingw64\\bin". Press ok.
+6. If you are using the MinGW compilers and dynamic (DLL) linking, add the path for the MinGW64 DLLs. Press NEW and enter "c:\\msys64\\mingw64\\bin". Press ok.
 7. Add the path for the EPICS commands and DLLs. Press NEW and enter ``%EPICS_BASE%\bin\%EPICS_HOST_ARCH%``. Press ok twice and you are done.
 8. Restart the Machine and check if EPICS commands like ``caget`` and ``camonitor`` are being recognised as valid commands in any location and work.
 
-Note that by default the MSYS2 shell does not inherit the parent environment. To change that behavior, you need to start the shell with the argument ``-use-full-path``.
+Note that by default the MSYS2 shell does not inherit the parent environment. To change that behavior, you need to start the shell with the argument ``-use-full-path``. To always start with the argument, replace ``Rem set MSYS2_PATH_TYPE=inherit`` with ``set MSYS2_PATH_TYPE=inherit`` in "c:\\msys64\\msys2_shell.cmd".
```

Comment:
What about...

To make the MSYS2 shell always inherit the parent environment, uncomment the line ``rem set MSYS2_PATH_TYPE=inherit`` in "C:\\msys64\\msys2_shell.cmd".

Reply:
Looks good. I will update